lib.include.sh

```diff
@@ -203,12 +204,25 @@ function has_conda_global_env {
 
 function create_conda_env {
     print "Creating Conda environment in \"${OT_CONDA_ENV}\"..."
+
     # IMPORTANT: The ".*" suffix tells Conda to install the latest bugfix/patch
     # release of the desired Python version. For example, if we specify "3.12.*",
     # then it will pick the latest patch release, such as "3.12.5". It also works
     # correctly if we specify an EXACT patch release ourselves, such as "3.10.14.*",
     # or if we only specify a major version, such as "3.*" (gets the latest release).
-    run_conda create -y --prefix "${OT_CONDA_ENV}" "python==${OT_CONDA_USE_PYTHON_VERSION}.*"
+    declare -a install_args=()
+    install_args+=("python==${OT_CONDA_USE_PYTHON_VERSION}.*")
```

Comment:
Can we merge these two lines? It's weird to declare something as empty and immediately append to it.

Reply:
I don't see an issue with this. Makes it explicit that this is meant to be a list where you can add new arguments by adding a new line

Reply:
@Nerogar Yeah, it's a code style I always use because it's explicit that it's an array.

Bash has a few different styles for declaring arrays.

The legacy way:

```sh
declare -a install_args
install_args+=("python==${OT_CONDA_USE_PYTHON_VERSION}.*")
install_args+=("tk[build=xft_*]")
```

The explicit declaration way (my preference):

```sh
declare -a install_args=()
install_args+=("python==${OT_CONDA_USE_PYTHON_VERSION}.*")
install_args+=("tk[build=xft_*]")
```

The "omg I don't like this, it looks weird and can lead to very long lines" way:

```sh
declare -a install_args=("python==${OT_CONDA_USE_PYTHON_VERSION}.*")
install_args+=("tk[build=xft_*]")
```